packages/experimental/Avatar/src/JSAvatar.tsx

```diff
@@ -29,33 +29,32 @@ export const JSAvatar = compose<JSAvatarType>({
   ...stylingSettings,
   slots: {
     root: View,
-    photo: Image,
+    image: Image,
     initials: Text,
     initialsBackground: View,
     icon: Image,
     ring: View,
-    glow: View,
+    badge: PresenceBadge,
   },
   useRender: (userProps: JSAvatarProps, useSlots: UseSlots<JSAvatarType>) => {
     const avatar = useAvatar(userProps);
     const Slots = useSlots(userProps, (layer) => avatarLookup(layer, avatar.state, userProps));
 
     return (final: JSAvatarProps) => {
-      const { children, accessibilityLabel, presence, activeAppearance, isOutOfOffice, ...mergedProps } = mergeProps(avatar.props, final);
-      const { personaPhotoSource, showRing, transparentRing } = avatar.state;
+      const { activeAppearance, children, image, badge, ...mergedProps } = mergeProps(avatar.props, final);
```

Comment:
It doesn't look like you're using children?

If you need children, you need to add another arg to the wrapper function (`return (final: JSAvatarProps, ...children: React.ReactNode[]) => {`) since staged components take out the children prop from the other props and pass it in as an arg to the inner render function. The children prop you take out of the result of mergeProps should be undefined

Reply:
Yes, you're right. It was from the old code of PersonaCoin. I'll fix it. Thank you for catching it